notebooks/05-operations/02-druid-logs-asynch.ipynb

```diff
@@ -0,0 +1,381 @@
+    "\n",
+    "Run this command to view the `common.runtime.properties` for the `start-druid` environment.\n",
+    "\n",
+    "```bash\n",
+    "cat ~/learn-druid-local/apache-druid/conf/druid/auto/_common/common.runtime.properties\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "28c0f9d0-9127-4d2c-a97e-d205fafae862",
+   "metadata": {},
+   "source": [
+    "### Viewing local logs\n",
+    "\n",
+    "The \"Indexing Service Logs\" section contains the configuration that caused the Middle Manager to write its logs to local disk:\n",
+    "\n",
+    "```\n",
+    "druid.indexer.logs.type=file\n",
+    "druid.indexer.logs.directory=var/druid/indexing-logs\n",
+    "```\n",
+    "\n",
+    "Run this command to see where task logs were written by looking at the Middle Manager log:\n",
+    "\n",
+    "```bash\n",
+    "grep \"Wrote task log to\" middleManager.log\n",
+    "```\n",
+    "\n",
+    "Notice that the log filename is made up of the query ID and the suffix `.log`.\n",
+    "\n",
+    "Adapt the following `vi` command to open the log for viewing:\n",
+    "\n",
+    "```bash\n",
+    "vi ~/learn-druid-local/apache-druid/<logfilelocation>\n",
+    "```\n",
+    "\n",
+    "For example,\n",
+    "\n",
+    "```bash\n",
+    "vi ~/learn-druid-local/apache-druid/var/druid/indexing-logsquery-83651faf-d179-4f4d-b0ad-ce61e36e85bd-worker0_0.log\n",
+    "```\n",
+    "\n",
+    "The logs contain a large amount of information helpful in understanding what Druid does during an ingestion. \n",
+    "\n",
+    "* The initial Java process spin-up.\n",
+    "* Records of activity, broken up into \"lifecycle stages\" (`org.apache.druid.java.util.common.lifecycle.Lifecycle')\n",
+    "* The actual creation of segment files and their publication.\n",
+    "* A JSON object containing the task completion status.\n",
+    "\n",
+    "Type `:q` and press enter to quit `vi`."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2e7474ae-cff4-47e4-b7ce-b80b786e3892",
+   "metadata": {},
+   "source": [
+    "### Viewing logs via the API\n",
+    "\n",
+    "Use the task log API with a query ID to access the logs over HTTP. This is the same as is used on the console to view logs.\n",
+    "\n",
+    "Switch `<queryID>` for the IDs output above, and then run it to pull the log file of one of the tasks.\n",
+    "\n",
+    "```bash\n",
+    "curl http://localhost:8888/druid/indexer/v1/task/<queryID>/log\n",
+    "```\n",
+    "\n",
+    "For example:\n",
+    "\n",
+    "```bash\n",
+    "curl http://localhost:8888/druid/indexer/v1/task/query-83651faf-d179-4f4d-b0ad-ce61e36e85bd-worker0_0/log\n",
+    "```\n",
+    "\n",
+    "Removing `-worker0_0` (and similar) will retrieve the overall log for the ingestion.\n",
+    "\n",
+    "Use the Tasks API to list all possible task IDs.\n",
```

Comment:
I think you should start the cell with this and then show the specific query log API, so that users have a query Id to use when they try individual logs.